docs/index.html

```diff
@@ -76,33 +76,33 @@ <h1 class='logo'>
 <a class='social__link' href='http://twitter.com/RubyConfCo' taget='_blank'>Twitter</a>
 </div>
 <div class='intro'>
-<h2 class='intro__title'>&gt;&gt; Hola ¿estás listo?</h2>
+<h2 class='intro__title'>&gt;&gt; Are you ready?</h2>
 <p class='intro__description'>
-Queremos que seas parte de <strong>RubyConf Colombia 2017</strong>. Nuestro equipo organizador está trabajando activamente en seleccionar el mejor contenido para hacer un evento inolvidable.
+Be part of <strong>RubyConf Colombia 2017</strong>. The organizing team is hard at work to select the best content and make this the best an experience the best Ruby conference in Colombia ever.
```

Comment:
I think it is better like this:
Be part of <strong>RubyConf Colombia 2017</strong>. The organizing team is working hard selecting the best content to organize the best Ruby conference in Colombia ever.